package.json

```diff
@@ -48,6 +48,7 @@
     "dotenv": "^8.2.0",
     "eslint-config-hackreactor": "github:reactorcore/eslint-config-hackreactor",
     "express": "^4.15.0",
+    "express-redis-cache": "^1.1.3",
```

Comment:
I would only keep around if you are going to use, but it looks like you are only interacting with `Redis` module directly